source/common/common/callback_impl.h

```diff
@@ -24,6 +24,7 @@ template <typename... CallbackArgs> class CallbackManager {
    */
   CallbackHandle* add(Callback callback) {
     callbacks_.emplace_back(*this, callback);
```

Comment:
it looks good. can you move this into the constructor? caller doesnt have to modify the handle outside of its box

Reply:
I am not sure what is going on, I will pull from upstream to retest it first, it the same error show up again, I will test the change  to find the bug.